.versionrc.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Personally prefer the command line flags version, but whatever suits you.

Reply:
Me too, but I couldn’t find a way to specify the packageFiles type through the command line so I had to revert to this abomination.